tooling-cpu-load/src/main/java/org/jboss/eap/qe/microprofile/tooling/cpu/load/HighCPUUtils.java

```diff
@@ -0,0 +1,66 @@
+package org.jboss.eap.qe.microprofile.tooling.cpu.load;
+
+import java.io.File;
+import java.time.Duration;
+
+import org.jboss.eap.qe.microprofile.tooling.cpu.load.utils.CpuLoadGenerator;
+import org.jboss.eap.qe.microprofile.tooling.cpu.load.utils.JavaProcessBuilder;
+import org.jboss.eap.qe.microprofile.tooling.cpu.load.utils.ProcessUtils;
+
+/**
+ * Utility class causing 100% CPU load on given CPU cores.
+ * <p>
+ * NOTE THIS UTILITY CLASS IS DEPENDENT ON RHEL/FEDORA SYSTEMS.
+ */
+public class HighCPUUtils {
+
+    /**
+     * Simulates that EAP/Wildfly server will be under 100% CPU load.
+     * <p>
+     * This method binds process with WF/EAP to given CPU cores and then causes 100% load on those cores so WF/EAP
+     * process has very little CPU time (almost not scheduled) on those cores.
+     *
```

Comment:
`containerProcessId` and `durationOfLoad` are missing

Reply:
thanks, fixed